docs/ensembl-help/using-ensembl/getting-ensembl-data/tools-and-apis/refget/refget.md

```diff
@@ -0,0 +1,246 @@
+
+When converting from Ensembl coordinates (1-start, fully-closed; sometimes called 1-based) you need only remove `1` from the start. The end remains the same. For example, given an Ensembl coordinate (1-based) 1:12714000**1**-127140001, this is represented as 1:12714000**0**-127140001 (0-based).
+
+### Using appropriate `Accept` content-types
+
+Refget is particular about the content-types it is provided. When requesting a sequence you must specify `plain/text` as the content type and when accessing sequence metadata you must specify `application/json`. Some URLs will fail when clicked on within a web browser due to their automated provision of Accept content types.
+
+### Why a metadata mentions multiple aliases for Ensembl sequences rather than just one
+
+The refget system effectively creates a non-redundant database of sequences. If the same sequence of characters appear in the same order they will result in the same checksum identifier being generated irrespective of molecule type (nucleotides versus amino acid residues).
+
+### Use of multiple identifiers for a sequence
+
+Refget's specification details two checksum algorithms which can be used to generate the identifier. These are a hex encoded `MD5` or a scheme called [`sha512t24u`](https://samtools.github.io/hts-specs/refget.html#refget-checksum-algorithm). A GA4GH identifier is generated by adding `ga4gh:SQ.` to a `sh512t24u` checksum e.g. `ga4gh:SQ.0qw_sn8Cl7OmMTFlukjFD2DUejW0T80Y` and `17dab79b963ccd8e7377cef59a54fe1c` represent the same sequence.
+
+## Third party refget clients
+
+You can use the [Python `refget`](https://pypi.org/project/refget/) library to fetch sequences from the Ensembl refget instance.
+
+## Accessing sequence data for features in Ensembl
+
+The ["Getting started with Graphql"](../tools-and-apis/graphql/using-graphql.md) and ["Using Graphql"](../tools-and-apis/graphql/using-graphql.md) pages cover how to access data through Ensembl GraphQL service and the example below assumes that this service is being used.
+
+Before starting you must obtain a `genome_id` for the genome (assembly + annotation) of interest. We are using `"GCA_000001405.29`, which represents the human assembly GRCh38.p14 (hg38).
+
+```text
+query {
+  genomes(
+    by_assembly_accession_id: {assembly_accession_id: "GCA_000001405.29"})
+  {
+    genome_id
+  }
+}
+```
+
+returns:
+
+```json
+{
+  "data": {
+    "genomes": [
+      {
+        "genome_id": "a7335667-93e7-11ec-a39d-005056b38ce3"
+      }
+    ]
+  }
+}
+```
+
+### Fetching the genomic sequence for a feature
+
+Using the previously retrieved `genome_id`, we can search for a feature (such as a gene) you myst find the checksum for a feature's (gene, transcript etc.) region, and its start and end coordinates via `slice`
+
+```text
+query {
+  genes(
+    by_symbol: {genome_id: "a7335667-93e7-11ec-a39d-005056b38ce3", symbol: "JAG1"}
```

Comment:
Do you think the users are more likely to query by symbol or by stable id? Query by symbol results in an array of genes (the `genes` field); but query by stable id returns a single gene (in a `gene` field), which might be easier on the reader.

Reply:
This was chosen so that it's more human readable.  UUID and a symbol seemed nicer than UUID and another ID.

Reply:
Going to stick with the symbol version.